controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -321,21 +304,18 @@ public void refresh() {
     }
 
     @Override
-    CompletableFuture<Void> createHistoryTable(Data<Integer> data) {
+    CompletableFuture<Void> createHistoryTableIfNotExists(Data<Integer> data) {
```

Comment:
This pattern of adding `NotExists` is confusing, I'm not entirely clear why we are changing all these names in here.